project/Common.scala

```diff
@@ -12,6 +12,10 @@ object Common {
   val scalaBinaryVersionNumber = "2.12"
   val scalaVersionNumber = s"$scalaBinaryVersionNumber.10"
 
+  val scala213 = "2.13"
```

Comment:
It would be nice to have the variable match the scala 2.12 one

```suggestion
  val scala213BinaryVersionNumber = "2.13"
```

Reply:
Done ✅